build-deb-package

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+#
+# Builds a Debian package for MXNET
+#
+# Copyright sociomantic labs GmbH 2017.
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file BOOST_LICENSE_1.0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+set -xeu
+
+# if the requested build identifier matches a git tag, use that directly,
+# otherwise use git revision description (note, we implicitly assume that all
+# tags and revision descriptions will begin with `v`)
+versiontag=$(git describe)
```

Comment:
Actually, I don't understand what you are referring to here.  The code you reference doesn't convert from SemVer to debian versioning, it just checks whether `ref` is a valid tag: if it is, it is used as the version, and if not, the output of `git describe` is used instead.

Is there something else that should be used here instead?